sky/cli.py

```diff
@@ -683,7 +683,8 @@ def _launch_with_confirm(
     if maybe_status is None:
         # Show the optimize log before the prompt if the cluster does not exist.
         try:
-            backend_utils.check_public_cloud_enabled()
+            if not is_local_cloud:
+                backend_utils.check_public_cloud_enabled()
```

Comment:
Curious - why don't we check only for public clouds and not all clouds? More specifically, why isn't this method just `backend_utils.check_cloud_enabled()`, which won't raise a runtime error when only local cloud is enabled? 

Could we change the definition of `enable` for `local` cloud to mean when one or more valid local cluster yaml is present in `~/.sky/local`?